llpc/tool/llpcCompilationUtils.h

```diff
@@ -95,6 +96,9 @@ struct CompileInfo {
   bool autoLayoutDesc;            // Whether to automatically create descriptor layout based on resource usages
   bool robustBufferAccess;        // Whether to enable robust buffer access
   bool scratchAccessBoundsChecks; // Whether to enable scratch access bounds checks
+#if LLPC_CLIENT_INTERFACE_MAJOR_VERSION > 52
+  llvm::CodeGenOpt::Level optimizationLevel; // The optimization level to pass the compiler
+#endif
```

Comment:
I think it'd be better is this was unconditional here. This `struct` is only used by amdllpc and XGL can't reference it.